src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java

```diff
@@ -16,15 +16,26 @@
  * Deletes a task identified using it's displayed index on display panel.
  */
 public class DeleteTaskCommand extends Command {
+
     public static final String COMMAND_WORD = "rmt";
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + "[display index] + \n"
-            + "Constraint: Task list must be shown on display panel using \"cat\" command. + \n"
-            + "Parameters: display index (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_USAGE =
+            "Usage: " + COMMAND_WORD + " <task index>\n"
+            + "\n"
+            + "Delete a task with the specified index number.\n"
+            + "\n"
+            + "Argument: \n"
+            + "    task index           Valid path to group or student\n"
+            + "\n"
+            + "Examples: \n"
+            + "rmt 1";
+
     public static final String MESSAGE_TASK_LIST_NOT_SHOWN = "Current display panel is not displaying task list.";
```

Comment:
Wrong description for task index, and also just make it short like "index", then desc: is valid display index for target task